fe/fe-core/src/main/java/com/starrocks/lake/compaction/ScoreSelector.java

```diff
@@ -32,7 +32,7 @@ public List<PartitionStatistics> select(@NotNull Collection<PartitionStatistics>
         return statistics.stream()
                 .filter(p -> p.getNextCompactionTime() <= now)
                 .filter(p -> p.getCompactionScore() != null)
-                .filter(p -> p.getCompactionScore().getAvg() >= minScore || p.getCompactionScore().getP50() >= minScore)
+                .filter(p -> p.getCompactionScore().getAvg() >= minScore || p.getCompactionScore().getMax() >= minScore)
```

Comment:
Why changed this?

Reply:
Because I find some tablets have higher compaction score while avarage score of partition is low. But the query latency depends on the tablets with high score. So we want to get low compaction score for all tablet s:)